Documentation/Testing/ExtensionTesting.rst

```diff
@@ -557,8 +557,8 @@ once in a while. A perfect scenario for a `functional test!
             $queryBuilder->getRestrictions()->removeAll();
             $count = $queryBuilder->count('uid')
```

Comment:
We should add a (int) cast here. It may be not an integer which is returned (for ex. with sqlite and PHP before 8.1)